### ISSUE TITLE
WPFTweaksLoc

### ISSUE THREAD

**Issue by Obegg** [ChrisTitusTech/winutil#1920]
Why would you do
https://github.com/ChrisTitusTech/winutil/blob/2354645b479b9ca7bcf377ea4e199b262f4b6bb4/config/tweaks.json#L80-L116

When you can do
```powershell
if((Test-Path -LiteralPath "HKLM:\Software\Policies\Microsoft\Windows\LocationAndSensors") -ne $true) {  New-Item "HKLM:\Software\Policies\Microsoft\Windows\LocationAndSensors" -force -ea SilentlyContinue };
New-ItemProperty -LiteralPath 'HKLM:\Software\Policies\Microsoft\Windows\LocationAndSensors' -Name 'DisableLocation' -Value 1 -PropertyType DWord -Force -ea SilentlyContinue;

```

**Comment by Marterich** [comment]
TLDR: **Maintainability** and **Error Handling & Logging** and **Scalability**

**Maintainability:**
It's easier to add/modify a new registry key if all the information you need to provide is simply supplied via an Object containing the Path, Name, Type, Value, and Original Value. Those values can then be used dynamically as a function argument to set the values. It is also way easier for code review to check if a new change e.g. a new registry key makes sense and all the values are provided if you simply have to check if the provided values are correct without needing to verify all the logic around changing or removing a key for every change.

**Error Handling & Logging**
The object of the registry key is handed over to a function that handles the actual changing of the key. The function (Set-WinUtilRegistry) works the same for _every_ registry key object in the script and does some error handling as well, like checking if the key even exists. Also, the central function writes output in a standardized way, so if you ever want to change the logic on how to set or remove registry keys, you would only have to edit a single file instead of every occurrence of a registry key change individually.
